### DISCUSSION TITLE
Cannot pass data to nested component

### DISCUSSION THREAD

**Discussion by TheRealFaruhaan** [livewire/livewire#4540] [1. Help]
I have two livewire components. One component is nested inside another component. When calling to the nested component, I am trying to pass a variable which I am unable to do so. I have done like these several times, but at the moment am unable.

Inside the **main component**'s blade file (**administration.groups-management**):

```
<div id="usersList" class="collapse mb-4 mt-1">
    {{ $selectedGroup }} <!-- for debug just to ensure it is not null -->
    @livewire('administration.groups-management.users-list', ['selectedGroup' => $selectedGroup])
</div>
```
Inside the **nested component**'s livewire class (**Administration>GroupsManagement>UsersList**):

```
class UsersList extends Component
{
    public $selectedGroup;
    public function render()
    {
        return view('livewire.administration.groups-management.users-list');
    }
}
```
And right now when I check, the *$selectedGroup* in the **groups-management** blade file is clearly **not** *null*, however, in the **nested livewire component**'s class, the *$selectedGroup* is *null*. Additionally, I have also tried mounting with mount method inside users list livewire class with:

```
public function mount($selectedGroup){
    $this->selectedGroup = $selectedGroup;
}
```
That doesn't work either. Please let me know what I am doing wrong here.

**Comment by joshhanley** [comment]
@TheRealFaruhaan all looks ok to me, that's how I do it. Not sure why it wouldn't be working for you. 

If you could create a small example repo which reproduces the issue with as little code as possible I'd be happy to have a look at it for you.
